cardano-testnet/src/Parsers/Cardano.hs

```diff
@@ -25,20 +26,23 @@ data CardanoOptions = CardanoOptions
 
 optsTestnet :: Parser CardanoTestnetOptions
 optsTestnet = CardanoTestnetOptions
-  <$> OA.option
-      ((`L.replicate` cardanoDefaultTestnetNodeOptions) <$> auto)
-      (   OA.long "num-bft-nodes"
-      <>  OA.help "Number of BFT nodes"
-      <>  OA.metavar "COUNT"
-      <>  OA.showDefault
-      <>  OA.value (cardanoBftNodeOptions defaultTestnetOptions)
-      )
-  <*> OA.option auto
-      (   OA.long "num-pool-nodes"
-      <>  OA.help "Number of pool nodes"
-      <>  OA.metavar "COUNT"
-      <>  OA.showDefault
-      <>  OA.value (cardanoNumPoolNodes defaultTestnetOptions)
+  <$> ((++)
+        <$> OA.option
+              ((`L.replicate` BftTestnetNodeOptions []) <$> auto)
+              (   OA.long "num-bft-nodes"
+              <>  OA.help "Number of BFT nodes"
+              <>  OA.metavar "COUNT"
+              <>  OA.showDefault
+              <>  OA.value (cardanoNodes defaultTestnetOptions)
+              )
+        <*> OA.option
+              ((`L.replicate` SpoTestnetNodeOptions) <$> auto)
+              (   OA.long "num-pool-nodes"
+              <>  OA.help "Number of pool nodes"
+              <>  OA.metavar "COUNT"
+              <>  OA.showDefault
+              <>  OA.value (cardanoNodes defaultTestnetOptions)
+              )
```

Comment:
Perhaps it's better to introduce a `mkCardanoTestnetOptions` function that takes the arguments exactly as we need them so we don't have to do nested parsing like this.

Reply:
The description still says `num-pool-nodes`, etc.

Reply:
Will the list argument to `BftTestnetNodeOptions` always be empty?